.github/workflows/test-and-build.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Test and build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x, 17.x]
```

Comment:
That would be nice, but it requires that we change this dependency: 
    "github-colors": "git+https://github.com/ozh/github-colors.git",
Since it breaks in Node 14.